### DISCUSSION TITLE
Is there a group chat for this community?

### DISCUSSION THREAD

**Discussion by nrvw** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#506] [General]
I'd like to share some strategies/results, would be nice if we can share these data via discord or something

**Comment by Yishaqel** [comment]
At the moment there is no official discord.

**Comment by Tkd-Alex** [comment]
Nope :) If someone wants to open a Discord channel It's free to do and the chanell will be magically the official one :magic_wand:

**Reply by tankow79** [reply]
Has the Discord channel already been created ?
